src/improvements/tasks/sep/020-worldchain-l1-ownership-transfers/VALIDATION.md

```diff
@@ -0,0 +1,111 @@
+# Validation
+
+This document can be used to validate the inputs and result of the execution of the upgrade transaction which you are signing.
+
+The steps are:
+1. [Validate the Domain and Message Hashes](#expected-domain-and-message-hashes)
+2. [Verifying the state changes via the normalized state diff hash](#normalized-state-diff-hash-attestation)
+3. [Verifying the transaction input](#understanding-task-calldata)
+4. [Verifying the state changes](#task-state-changes)
+
+## Expected Domain and Message Hashes
+
+First, we need to validate the domain and message hashes. These values should match both the values on your ledger and the values printed to the terminal when you run the task.
+
+> [!CAUTION]
+>
+> Before signing, ensure the below hashes match what is on your ledger.
+>
+> ### Worldchain Proxy Admin Owner (`0x945185C01fb641bA3E63a9bdF66575e35a407837`)
+>
+> - Domain Hash:  `0x6faec9c52949ba8274340008df12c69faedd5c44e77f77c956d2ca8e4bcd877e`
+> - Message Hash: `0x55faea00e974897c5afad9bdece6e4239df3b44a54dc03bab2f9172c5ab91c9c`
+
+## Normalized State Diff Hash Attestation
+
+The normalized state diff hash **MUST** match the hash produced by the state changes attested to in the state diff audit report. As a signer, you are responsible for verifying that this hash is correct. Please compare the hash below with the one in the audit report. If no audit report is available for this task, you must still ensure that the normalized state diff hash matches the output in your terminal.
+
+**Normalized hash:** `0x468b869284297f167bc8a9e7589ccbd096c1fab043792a6e461b5ad8d30d556b`
+
+## Understanding Task Calldata
+
+The commands to encode the calldata is:
+
+First lets define all the contracts that will have their ownership transferred:
+
+- DisputeGameFactoryProxy: [`0x8Ec1111f67Dad6b6A93B3F42DfBC92D81c98449A`](https://github.com/ethereum-optimism/superchain-registry/blob/1ff0df40c7602761c55ab2cb693614ca0382bd64/superchain/configs/sepolia/worldchain.toml#L64C30-L64C72)
+
+- Permissioned DelayedWETHProxy: [`0xAEB3CfD5aAba01cfd12E6017a9a307a218cdD7E2`](https://sepolia.etherscan.io/address/0x552334Bf0B124bD89BFF744f33Ca7e49d44a80Ac#readContract#F37)
+
+- ProxyAdmin: [`0x3a987FE1cb587B0A1808cf9bB7Cbe0E341838319`](https://github.com/ethereum-optimism/superchain-registry/blob/1ff0df40c7602761c55ab2cb693614ca0382bd64/superchain/configs/sepolia/worldchain.toml#L60C17-L60C59)
```

Comment:
This section is only partially complete. If it's not going to be fully filled out for this task lets just remove it.
```suggestion
```